android/src/main/kotlin/io/lakscastro/sharedstorage/storageaccessframework/DocumentFileApi.kt

```diff
@@ -311,6 +318,25 @@ internal class DocumentFileApi(private val plugin: SharedStoragePlugin) :
     }
   }
 
+  private fun writeToFile(
+    result: MethodChannel.Result,
+    uri: String,
+    content: ByteArray,
+    mode: String,
```

Comment:
Syntax error because of this comma:

> **Warning** The feature "trailing commas" is only available since language version 1.4

<kbd>
  <img src="https://user-images.githubusercontent.com/51419598/179240360-e3ba6304-acd3-4208-bc0e-0fd6bbf598eb.png">
</kbd>